app/views/simulator/edit.html.erb

```diff
@@ -132,22 +132,22 @@
           </div>
           <div style="position:absolute; right:60px; top:30px; z-index:100">
             <button type="button" class="btn zoomButton btn-lg" onclick="globalScope.centerFocus(false)">
-              <i class="fa fa-arrows" aria-hidden="true"></i>
+              <span class="fa fa-arrows" aria-hidden="true" title="Center Focus"></span>
```

Comment:
Can you rename `center focus` it to `zoom to fit` or `center canvas` . Also not sure why its not working for me 
![image](https://user-images.githubusercontent.com/2092958/76168143-c0716d80-6192-11ea-9bba-62d5966a4dfb.png)